### ISSUE TITLE
Consider allowing to install suggested package without the package suggesting it

### ISSUE THREAD

**Issue by m-vo** [contao/contao-manager#653]
see https://github.com/contao/contao/pull/3228#issuecomment-890332730

**Comment by aschempp** [comment]
I don't think this is how suggestions are supposed to work. In a regular package setup, suggestions are something that add additional features to the package if available. For example in Isotope, [we suggest payment gateway APIs](https://github.com/isotope/core/blob/2.7/composer.json#L56-L62) because the payment methods are implemented in Isotope, and if you install the API library they will be available automatically (simply through `class_exists`).

I have created a pull request at https://github.com/twigphp/twig-extra-bundle/pull/3 to add suggestions to `twigphp/twig-extra-bundle`. If the PR is approved, we can add `twigphp/twig-extra-bundle` to our search index manually. If the package is installed (which is the default case in Contao 4.12), people can install the suggestions. It will also allow to install the bundle in Contao < 4.12 which will not work out-of-the-box, but maybe thats a limitation we can live with...

**Comment by mlwebworker** [comment]
Moritz das war fast nur für Dich.
